bika/lims/content/sample.py

```diff
@@ -760,13 +747,19 @@ def setClientReference(self, value, **kw):
             ar.Schema()['ClientReference'].set(ar, value)
         self.Schema()['ClientReference'].set(self, value)
 
+    def getClientReference(self):
+            return self.Schema().getField('ClientReference').get(self)
+
     def setClientSampleID(self, value, **kw):
         """ Set the field on Analysis Requests.
         """
         for ar in self.getAnalysisRequests():
             ar.Schema()['ClientSampleID'].set(ar, value)
         self.Schema()['ClientSampleID'].set(self, value)
 
+    def getClientSampleID(self):
+        return self.Schema().getField('ClientSampleID').get(self)
```

Comment:
since there is a `ClientSampleID` field defined in the Schema already, why the creation of a getter is still required (plone automatically generates the getter for each Schema field at runtime)?